examples/scripts/project/_common/fork_project.py

```diff
@@ -77,13 +77,8 @@ def main(
     LOGGER.info(op)
 
     ## preparing for the import
-    # finding the path to export file
-    zipfile_name = [
-        f
-        for f in os.listdir(export_path)
-        if (os.path.isfile(os.path.join(export_path, f)) and f.endswith(".zip"))
-    ][0]
-    zipfile_path = f"{export_path}/{zipfile_name}"
+    # finding the path to export file from op JSON response
+    zipfile_path = op.description.split(":")[-1].strip()
```

Comment:
I'm concerned that this will not handle cases where a `:` appears in the export path.  Rather than splitting and taking the last element, can you figure out a way for this to always just trim off the first `": "` and return the rest of the message?

Reply:
How about something like this @skalish :

![image](https://user-images.githubusercontent.com/77649523/190485322-a2dc2a62-dd9d-4e67-87c5-972da402faf1.png)


Is it fair to assume that description always starts with: `projectImport with artifact: `?

Reply:
Or maybe we can do something more simplified less precise such as: 
![image](https://user-images.githubusercontent.com/77649523/190485900-62a4b939-3b84-4722-81ab-1866b5a1cb53.png)